ci/infra/testrunner/kubectl/kubectl.py

```diff
@@ -12,6 +19,22 @@ def __init__(self, conf, platform):
         self.platform = platforms.get_platform(conf, platform)
         self.skuba = Skuba(conf, platform)
 
+    @timeout(300)
```

Comment:
I think this function shouldn't be in kubectl class but as part of the testlib. We should keep a clear separation of concerns.